modules/ROOT/pages/sync-update-api-spec.adoc

```diff
@@ -0,0 +1,22 @@
+= Update the API Specification in Your Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+When necessary you can update the API specification in your Studio project.
+
+To update the API specification:
+
+. In the Package Explorer view, right-click the Mule project that contains the API specification you want to update.
+. Select *API Specs* > the name of your API > *Update Version*.
++
+After you update the specification, Studio asks you if you want to regenerate the flows based on the new version.
++
+To manually generate a flow for the updated API, right-click the Mule project where you updated the API specification, and select *API Specs* > the name of your API > *Generate Flows*.
+
+Additionally, you can manage the API specifications in your project that are linked to Exchange from the API specification project management view. To do so, click the *Manage Mule Project APIs* icon (image:manage-project-api-icon.png[2%,2%]) in the top task bar.
+
+image::update-api-version.png[]
```

Comment:
There is a padding missing in this screenshot.

Reply:
uploaded new image